Gemfile

```diff
@@ -1,14 +1,16 @@
-source 'https://rubygems.org'
+# frozen_string_literal: true
+
+source "https://rubygems.org"
 gemspec
 
-if Dir.exist? 'jekyll/docs'
-  require 'yaml'
-  config = YAML.load_file 'jekyll/docs/_config.yml'
+if Dir.exist? "jekyll/docs"
+  require "yaml"
+  config = YAML.load_file "jekyll/docs/_config.yml"
 
   gems = []
-  gems.concat config['gems'] if config.include? 'gems'
-  gems.concat config['plugins'] if config.include? 'plugins'
-  gems << 'pygments.rb' if config['highlighter'] == 'pygments'
-
+  gems.concat config["gems"] if config.include? "gems"
+  gems.concat config["plugins"] if config.include? "plugins"
+  gems << "pygments.rb" if config["highlighter"] == "pygments"
+  gems << "rubocop"
```

Comment:
Is this the right way to add the gem. Travis is failing 😕

Reply:
No, this block is only evaluated if jekyll is cloned. Add `gem “rubocop”` above `if Dir.exist`